po/de.po

```diff
@@ -0,0 +1,155 @@
+msgid ""
+"Flatseal is a graphical utility to review and modify basic permissions from "
+"your Flatpak applications."
+msgstr ""
+"Flatseal ist ein grafisches Dienstprogramm zum Überprüfen und Ändern von Basisberechtigungen "
+"Ihre Flatpak-Anwendungen."
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:29
+msgid "Martin Abente Lahaye"
+msgstr ""
+
+#: src/aboutDialog.ui:11
+msgid "Copyright © 2020 - Martín Abente Lahaye"
+msgstr ""
+
+#: src/applicationRow.ui:57
+msgid "Application"
+msgstr "Anwendungen"
+
+#: src/applicationRow.ui:74 src/permissionEntryRow.ui:45
+#: src/permissionSwitchRow.ui:45
+msgid "Description"
+msgstr "Beschreibung"
+
+#: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
+msgid "Pemission"
+msgstr "Berechtigung"
+
+#: src/view.ui:36
+msgid "No applications found."
+msgstr "Keine Anwendungen gefunden."
+
+#: src/window.ui:41
+msgid "Reset"
+msgstr "Zurücksetzen"
+
+#: src/window.ui:47
+msgid "Reset this application permisions"
+msgstr "Berechtigung dieser Anwendung zurücksetzen"
+
+#: src/model.js:35
+msgid "Access network"
+msgstr "Zugriff auf das Netzwerk"
+
+#: src/model.js:40
+msgid "Access inter-process communications"
+msgstr "Zugriff zu die prozessübergreifende Kommunikation"
+
+#: src/model.js:45
+msgid "Access X11 windowing system"
+msgstr "Zugriff auf den X11-Fenstermanager"
+
+#: src/model.js:50
+msgid "Access X11 windowing system (as fallback)"
+msgstr "Zugriff auf den X11-Fenstermanager (als Fallback)"
+
+#: src/model.js:55
+msgid "Access Wayland windowing system"
+msgstr "Zugriff auf den Wayland-Fenstermanager"
+
+#: src/model.js:60
+msgid "Access PulseAudio sound server"
+msgstr "Zugriff auf den PulseAudio-Soundserver"
+
+#: src/model.js:65
+msgid "Access D-Bus system bus (unrestricted)"
+msgstr "Zugriff auf den D-Bus-Systembus (uneingeschränkt)"
+
+#: src/model.js:70
+msgid "Access D-Bus session bus (unrestricted)"
+msgstr "Zugriff auf den D-Bus-Sitzungsbus (uneingeschränkt)"
+
+#: src/model.js:75
+msgid "Access Secure Shell agent"
+msgstr "Zugriff auf den Secure Shell Agent"
+
+#: src/model.js:80
+msgid "Access printing system"
+msgstr "Zugriff auf die Druckerverwaltung"
+
+#: src/model.js:85
+msgid "Access GPU acceleration"
+msgstr "Zugriff auf GPU-Beschleunigung"
+
+#: src/model.js:90
+msgid "Access all devices (e.g. webcam)"
+msgstr "Zugriff auf alle Geräte (z.B. Webcam)"
+
+#: src/model.js:95
+msgid "Access all system directories (unrestricted)"
+msgstr "Zugriff auf alle Systemverzeichnisse (uneingeschränkt)"
+
+#: src/model.js:100
+msgid "Access home directory (unrestricted)"
+msgstr "Zugriff auf das Benutzerverzeichnis (uneingeschränkt)"
+
+#: src/model.js:105
+msgid "Access bluetooth"
+msgstr "Zugriff auf Bluetooth"
+
+#: src/model.js:110
+msgid "Access other syscalls (e.g. ptrace)"
+msgstr "Zugriff auf andere Systemaufrufe (z.B. ptrace)"
+
+#: src/model.js:115
+msgid "Access programs from other architectures"
+msgstr "Zuriff auf Programme anderer Architekturen"
```

Comment:
I noticed while testing `Zuriff`, is this a typo?

Reply:
Oh, you're right. It's a typo. It should be called "Zugriff" with a 'g'.